content/pt-br/blog/_posts/2023-02-14-free-katacoda-kubernetes-tutorials-are-shutting-down.md

```diff
@@ -0,0 +1,17 @@
+evergreen: true
+---
+
+**Author**: Natali Vlatko, SIG Docs Co-Chair para Kubernetes
+
+[Katacoda](https://katacoda.com/kubernetes), a popular plataforma de aprendizagem da O'Reilly que tem ajudado as pessoas a aprender tudo sobre Java, Docker, Kubernetes, Python, Go, C++ e mais, [foi desativada para uso público em junho de 2022](https://www.oreilly.com/online-learning/leveraging-katacoda-technology.html). No entanto, os tutoriais específicos para Kubernetes, vinculados no website do Kubernetes para os usuários e contribuidores de nossos projetos, permaneceram disponíveis e ativos após esta mudança. Infelizmente, isso não será mais o caso, e os tutoriais do Katacoda para aprender o Kubernetes deixarão de funcionar após 31 de março de 2023.
+
+O Projeto Kubernetes agradece à O'Reilly Media pelos muitos anos em que apoia a comunidade por meio da plataforma de aprendizado Katacoda. Você pode ler mais sobre [a decisão de fechar o katacoda.com](https://www.oreilly.com/online-learning/leveraging-katacoda-technology.html) no próprio website da O'Reilly. Com essa mudança, focaremos no trabalho necessário para remover os links para seus vários tutoriais. Temos uma Questão Geral para rastrear este tópico em [#33936](https://github.com/kubernetes/website/issues/33936) e [discussão no GitHub](https://github.com/kubernetes/website/discussions/38878). Também estamos interessados em pesquisar quais outras plataformas de aprendizado podem ser benéficas para a comunidade do Kubernetes, substituindo o Katacoda por uma plataforma ou um serviço que tenha uma experiência de usuário semelhante. No entanto, esta pesquisa levará tempo, por isso estamos procurando ativamente por voluntários para ajudarem neste trabalho. Se uma substituição for encontrada, ela precisará ser suportada pela liderança do Kubernetes, especificamente, SIG Contributor Experience, SIG Docs e Kubernetes Steering Committee.
+
+O desligamento do Katacoda afeta 25 páginas de tutorial, suas localizações, bem como o repositório de cenários do Katacoda: [github.com/katacoda-scenarios/kubernetes-bootcamp-scenarios](https://github.com/katacoda-scenarios/kubernetes-bootcamp-scenarios). Recomendamos que quaisquer links, guias ou documentação que apontem para a plataforma de aprendizado Katacoda sejam atualizados imediatamente para refletir essa alteração. Embora ainda não tenhamos encontrado uma solução de aprendizado substituta, o website do Kubernetes contém muita documentação útil para dar suporte ao seu aprendizado e crescimento contínuos. Você pode encontrar toda a documentação dos nossos tutoriais disponível para Kubernetes em <https://k8s.io/pt-br/docs/tutorials/>.
+
+Se você tiver alguma dúvida sobre o desligamento do Katacoda ou a remoção subsequente do link das páginas de tutorial do Kubernetes, sinta-se à vontade para comentar no [problema geral que acompanha o desligamento](<https://github.com/kubernetes/website/issues/33936>), ou visite o canal #sig-docs no Slack do Kubernetes.
```

Comment:
Same consideration about issue here. The chevrons in this link are also surplus and can be removed.

Reply:
the chevrons in full URL is required in the Markdown when it is not a "href".  But I removed from the hrefs. https://github.com/DavidAnson/markdownlint/blob/v0.27.0/doc/md034.md

Reply:
I don't think that's true for the flavour of Markdown that Hugo supports, but I was referring specifically to your link on line 17. Line 15 is alright.